docs/pages/branding/careers.tsx

```diff
@@ -0,0 +1,491 @@
+                "Excellence. We're aiming high, and we know it.",
+              ].map((text) => (
+                <Box key={text} sx={{ display: 'flex', alignItems: 'flex-start', mt: 1 }}>
+                  <IconImage name="yes" />
+                  <Typography variant="body2" color="text.primary" fontWeight={600} sx={{ ml: 1 }}>
+                    {text}
+                  </Typography>
+                </Box>
+              ))}
+            </Grid>
+            <MuiStats />
+          </Grid>
+        </Container>
+      </Box>
+      {/* Perks & benefits */}
+      <Box>
+        <Container sx={{ py: { xs: 4, md: 8 } }}>
+          <Grid container alignItems="center" spacing={4}>
+            <Grid item xs={12} md={6}>
+              <Typography variant="h2" sx={{ my: 1 }}>
+                {'Perks & benefits'}
+              </Typography>
+              <Typography color="text.secondary" sx={{ mb: 2 }}>
+                To help you go above and beyond with us, we provide:
+              </Typography>
+              {[
+                ['Remote work:', 'Our entire company is distributed.'],
+                [
+                  'Gatherings:',
+                  'We meet up once or twice a year for a short week of meetings, events, and fun!',
+                ],
+                [
+                  'Equipment:',
+                  'MUI will let you choose new hardware of your choice (up to $2,500 USD).',
+                ],
+                ['Time off:', 'We provide five weeks of paid time off.'],
+              ].map((textArray) => (
+                <Box key={textArray[0]} sx={{ display: 'flex', alignItems: 'flex-start', mt: 1 }}>
+                  <IconImage name="yes" />
+                  <Typography variant="body2" color="text.primary" sx={{ ml: 1 }}>
+                    <span style={{ fontWeight: 600 }}>{`${textArray[0]}  `}</span>
+                    {textArray[1]}
+                  </Typography>
+                </Box>
+              ))}
+            </Grid>
+            <Grid item xs={6} md={3}>
+              <Widget
+                title="Handbook"
+                url={'https://www.notion.so/mui-org/Handbook-f086d47e10794d5e839aef9dc67f324b'}
+              >
+                <Typography variant="body2" color="text.secondary" sx={{ mb: 1 }}>
+                  See how we run the company and the way we work.
+                </Typography>
+              </Widget>
+            </Grid>
+            <Grid item xs={6} md={3}>
+              <Widget title="Blog" url={'https://medium.com/material-ui'}>
+                <Typography variant="body2" color="text.secondary" sx={{ mb: 1 }}>
+                  Check behind the scenes and news from the company.
+                </Typography>
+              </Widget>
+            </Grid>
+          </Grid>
+        </Container>
+      </Box>
+      {/* Open roles */}
+      <Container sx={{ py: { xs: 4, md: 8 } }}>
```

Comment:
I feel like we could try a bigger container width when in Desktop. Just a little bit more. The text ended up a bit too crammed even though we have a lot of room:

<img width="400" alt="Screen Shot 2021-09-06 at 10 46 42" src="https://user-images.githubusercontent.com/67129314/132226899-3b2f65a7-afc7-42e6-b9bb-c6dc121b87e9.png">
<img width="400" alt="Screen Shot 2021-09-06 at 10 45 26" src="https://user-images.githubusercontent.com/67129314/132226905-2974a15c-995c-48a3-a7a6-1eaeffd01f4a.png">

Reply:
I changed max-width to 700px from 450px! Let me know!

Reply:
Looks great!